src/hooks/useIsAllowedToCreateDocuments.ts

```diff
@@ -0,0 +1,40 @@
+import { atom, useAtom } from 'jotai';
+import { useEffect } from 'react';
+import { useLogin } from './useLogin';
+
+const isAllowedAtom = atom<boolean>(false);
+
+export function useIsAllowedToCreateDocuments() {
+    const [isAllowed, setIsAllowed] = useAtom(isAllowedAtom);
+    const { user, status } = useLogin();
+    const createDocumentAllowListEnvString = import.meta.env
+        .VITE_CREATE_DOCUMENT_ALLOW_LIST;
+    const createDocumentAllowList =
+        createDocumentAllowListEnvString?.split(',');
+
+    useEffect(() => {
```

Comment:
Is the useEffect needed?

Reply:
I did it like this so it would react to changes in the user or auth status, e.g. if the user logs out and signs in with a different address. should we rather not?

Reply:
But we can do it outside of the useEffect or does it cause a loop?